src/ostype.rs

```diff
@@ -51,12 +52,13 @@ impl FromStr for OsType {
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         let lower = s.to_lowercase();
         match lower.as_str() {
+            "linux" => Ok(OsType::Linux),
+            "darwin" => Ok(OsType::Darwin),
```

Comment:
I think you can match both `darwin` and `macos` in the same branch: `"darwin" | "macos" => Ok(OsType::Darwin)`